05_syntax_lab/03.py

```diff
@@ -1,7 +1,6 @@
-"""
-Write a program that randomizes a number
-and prints the sum total of its digits.
-For example if the number was: 2345
-The result should be: 14
-"""
-
+#Week01_Exercise01_Assignment03
+from random import randint
+SumDigits = 0
+x = "%d" % randint(1,10000)
+for k in range(0,len(x)) : SumDigits += int(x[k])

```

Comment:
This works, and you can even remove the `range` and iterate over the actual characters (do try)